Add Rich Text Editor to 4 Quickstarts (#269)
Azure-Samples/communication-services-javascript-quickstarts: ui-library-quickstart-composites/README.md

![Composite End State](../media/CompositeEnd.png)

Feel free to style the composites to your desired size and layout inside of your application.

## Enable Rich Text Editor
 By default, the chat composite is using the plain text editor for the SendBox component and the MessageThread component's edit function.

>  We also provide a Rich Text Editor which provides rich text formatting, table inserting etc.

Reviewer: ```suggestion
 We also provide a Rich Text Editor for rich text formatting, table inserting etc.
```
